help/de/Help_ch_Customize.xml

```diff
@@ -1,2481 +1,3378 @@
+                    zu öffnen, in dem Sie die Schriftart für den Scheckdruck anpassen können.
+                  </para>
+                </listitem>
+              </varlistentry>
+            </variablelist>
+          </listitem>
+        </varlistentry>
+      </variablelist>
     </sect2>
 
     <sect2 id="prefs-reg">
-      <title>Register</title>
+      <title>Kontobuch</title>
 
       <figure id="prefs-reg-fig">
-        <title>The <guimenuitem>Preferences</guimenuitem> window, <guilabel>Register</guilabel> tab</title><screenshot id="Prefer_Register">
+        <title>Die &app;-<guimenuitem>Einstellungen</guimenuitem> für das
+          <guilabel>Kontobuch</guilabel></title><screenshot id="Prefer_Register">
           <mediaobject>
             <imageobject role="html">
               <imagedata fileref="figures/Help_Pref_Register.png"
-                         format="PNG" srccredit="Christian Wehling" width="510px"></imagedata>
+                         srccredit="Christian Wehling" width="&img-w;"></imagedata>
             </imageobject>
 
             <imageobject role="fo">
-              <imagedata fileref="figures/Help_Pref_Register.png" format="PNG"
+              <imagedata fileref="figures/Help_Pref_Register.png" 
                          srccredit="Christian Wehling"></imagedata>
             </imageobject>
 
-            <textobject>
-              <phrase>The Register tab</phrase>
-            </textobject>
+            <caption>
+              <para>Der Reiter <guilabel>Kontobuch</guilabel>
+              </para>
+            </caption>
           </mediaobject>
         </screenshot>
       </figure>
 
-      <itemizedlist>
-        <listitem>
-          <para><guilabel>Actions</guilabel>
-          </para>
+      <para>In diesem Bereich wird die Arbeit mit Buchungen sowie deren
+        Darstellung beeinflusst.
+      </para>
 
-          <itemizedlist>
-            <listitem>
-              <para><guilabel>'Enter' moves to blank transaction:</guilabel> If selected, move the cursor to the blank
-                transaction at the end of the register after the user presses the
-                <keycap function="enter">Enter</keycap> key. Otherwise the cursor will be moved down
-                one row.
-              </para>
-            </listitem>
+      <variablelist spacing="compact">
+        <varlistentry>
+          <term><guilabel>Aktionen</guilabel></term>
 
-            <listitem>
-              <para><guilabel>Auto-raise lists:</guilabel> If selected, all lists will automatically be expanded when
-                input focus is in the list field.
-              </para>
-            </listitem>
+          <listitem>
+            <variablelist spacing="compact">
+              <varlistentry>
+                <term><guilabel>Mit <quote>Eingabe</quote> gehen Sie zur leeren Buchung</guilabel></term>
```

Comment:
FAILED: help/de/CMakeFiles/de-gnucash-help-check /home/runner/work/gnucash-docs/gnucash-docs/build/_cmake_build/help/de/CMakeFiles/de-gnucash-help-check 
cd /home/runner/work/gnucash-docs/gnucash-docs/build/_cmake_build/help/de && /usr/bin/xmllint --postvalid --xinclude --noout --path /home/runner/work/gnucash-docs/gnucash-docs/build/gnucash-docs-4.6/docbook /home/runner/work/gnucash-docs/gnucash-docs/build/gnucash-docs-4.6/help/de/gnucash-help.xml
Help_ch_Customize.xml:1555: element guilabel: validity error : Element quote is not declared in guilabel list of possible children

Reply:
ich nutze den Befehl `xmllint` direkt, habe mich jedoch in der letzten Zeit schon gewundert, dass ich so fehlerfrei schreiben kann.
Dann gewöhne ich mir mal an, vor dem `Push` den `check` zu machen.

Reply:
Spätestens, wenn auf dieser Seite nach einiger Zeit neben dem commit ein rotes X statt eines grünen Häckchens auftaucht, sollten die Alarmglocken schrillen, Dann kann man im Reiter checks nachsehen, was schief gelaufen ist.